<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>391</width>
     <height>421</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="table">
    <attribute name="title">
     <string>  课程</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>381</width>
       <height>361</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="sunday">
      <attribute name="title">
       <string> 周日</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget_7">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>371</width>
         <height>311</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_0"/>
      </widget>
     </widget>
     <widget class="QWidget" name="monday">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string> 周一</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>371</width>
         <height>311</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_1"/>
      </widget>
     </widget>
     <widget class="QWidget" name="tuesday">
      <attribute name="title">
       <string> 周二</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>371</width>
         <height>311</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2"/>
      </widget>
     </widget>
     <widget class="QWidget" name="wednesday">
      <attribute name="title">
       <string> 周三</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>371</width>
         <height>311</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3"/>
      </widget>
     </widget>
     <widget class="QWidget" name="thursday">
      <attribute name="title">
       <string> 周四</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>371</width>
         <height>311</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_4"/>
      </widget>
     </widget>
     <widget class="QWidget" name="friday">
      <attribute name="title">
       <string> 周五</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>371</width>
         <height>311</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_5"/>
      </widget>
     </widget>
     <widget class="QWidget" name="saturday">
      <attribute name="title">
       <string> 周六</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget_6">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>371</width>
         <height>331</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_6"/>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>370</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>应用</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="info">
    <attribute name="title">
     <string>关于</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>150</y>
       <width>111</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>检查更新</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>350</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>   张家口市第一中学2017级16班 2019 版权所有 </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>  版本号 2.0.5</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>221</width>
       <height>71</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;开机启动请在注册表HKEY_LOCAL_MACHINE/SOFTWARE/Microsoft/Windows/CurrentVersion/Run中设置&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>440</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string> 退出</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
